src/components/common/AnnouncementBanner.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react'
+import { ReactComponent as MegaphoneIcon } from '../../assets/icons/ic-megaphone.svg'
+import { InfoColourBar } from '@devtron-labs/devtron-fe-common-lib'
+import { ReactComponent as Close } from '../../assets/icons/ic-close.svg'
+import { getDateInMilliseconds } from '../apiTokens/authorization.utils'
+import { setActionWithExpiry } from './helpers/Helpers'
+
+export default function AnnouncementBanner({ parentClassName = '', isCDMaterial = false }) {
+ 
+    const message = window?._env_?.ANNOUNCEMENT_BANNER_MSG
+    const showAnnouncementBanner = (): boolean => {
+        const expiryDateOfHidingAnnouncementBanner: string = localStorage.getItem(
+            //it will store date and time of next day i.e, it will hide banner until this date
+            'expiryDateOfHidingAnnouncementBanner',
+        )
+        const showAnnouncementBannerNextDay: boolean =
+            typeof Storage !== 'undefined' &&
```

Comment:
why this is only required here and not on line no 12?

Reply:
Added